fc/mef.py

```diff
@@ -607,14 +353,11 @@ def get_transform_fxn(data_beads, peaks_mef, mef_channels,
     ----------
     data_beads : FCSData object
```

Comment:
Can `data_beads` be a vanilla numpy array? Populating the plot filename from `str(data_beads)` seems like it would be a bad idea for a vanilla numpy array. I'm not even sure it's wise for an `FCSData` object (see discussion in #129 regarding `__repr__` and `__str__`).